.coveragerc

```diff
@@ -1,6 +1,8 @@
 [run]
 source = qutip
 omit =
+    # Skip dev tools
```

Comment:
I don't think we should skip the dev tools in coverage. It's always good to know what code is being covered by tests, even if we decide not to cover them.

Aside: I also think that tests *should* be included in coverage. I've often picked up issues where a test or part of a test isn't being run by looking at coverage (e.g. the test is skipped if a dependency isn't installed and no test runner instance has that dependency).